modules/network-observability-enriched-flows-kafka.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// network_observability/understanding-network-observability-operator.adoc
+
+:_content-type: Procedure
+[id="network-observability-enriched-flows-kafka_{context}"]
+= Export enriched network flow data
+
+You can send network flows to Kafka, so that they can be consumed by any processor or storage that supports Kafka input, such as Splunk, Elasticsearch, or Fluentd,
+
+.Prerequisites
+* Installed Kafka
+
+.Procedure
+
+. Edit the `FlowCollector` to configure `spec.exporters` as follows:
++
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  exporters:
+  - type: KAFKA
+      kafka:
+        address: "kafka-cluster-kafka-bootstrap.netobserv"
+        topic: netobserv-flows-export   <1>
+----
+<1> NetObserv exports all flows to the configured Kafka topic.
+
+. Next, flows data can be sent to an available output in a JSON format. 
```

Comment:
What about providing information on how this can be done in Web Console and the FlowCollector resource?

Reply:
@stleerh Is there a way users can export flows using the Web Console?

Reply:
There are a few parameters in the `kafka` section when you create FlowCollector.

Reply:
When you create a FlowCollector resource, you are able to configure this in the form view.  Do we want to show this?

Reply:
I think the opportunity to show this would be in the Installation section, where we have the list of items to select in form view. We could add an optional list item there.